app/components/member/Teams/chooser.js

```diff
@@ -168,7 +168,7 @@ class Chooser extends React.Component {
 
 export default Relay.createContainer(Chooser, {
   initialVariables: {
-    showingDialog: false,
+    isMounted: false,
```

Comment:
Why did you change this to `isMounted`? This _doesn’t_ occur when the dialog is mounted